.github/actionlint.yaml

```diff
@@ -0,0 +1,4 @@
+self-hosted-runner:
```

Comment:
We have prioritized the self-hosted runners to run the workflows that absolute need them. This check is likely something we can use standard GH runners to run.